src/kivy_state.py

```diff
@@ -0,0 +1,50 @@
+"""
+Kivy_state.py
+=================================
+"""
+import collections
+
+
+# Network protocols availability, initialised below
+networkProtocolAvailability = None
```

Comment:
make this into a class and instantiate one inside the `__init__` of `KivyStateVariables`